docs/getting-started/deployment.md

```diff
@@ -168,9 +168,9 @@ time-consuming process, we provide pre-generated material.
 1. Download and decompress the archive containing the material using:
 
     ```shell
-    curl -O -L https://github.com/carbynestack/base-images/raw/3595c5427915b2f9e1f22804e3f742cda9e72312/fake-crypto-material.zip
-    unzip -d crypto-material fake-crypto-material.zip
-    rm fake-crypto-material.zip
+    curl -O -L https://github.com/carbynestack/carbynestack/raw/master/tuples/fake-crypto-material-0.2.zip
```

Comment:
Please replace `master` with a concrete commit sha to lower the risk of breaking the tutorial when updating `master`.

Reply:
Done